base/install-functions.sh

```diff
@@ -36,14 +36,36 @@ install_module() {
     cd -
 }
 
+install_ioc() {
+    ioc_name=$1
+    dependency_name=$2
+    release_modules="$3"
+
+    cd $ioc_name
+    get_module_path "$release_modules" > configure/RELEASE
+
+    make -j${JOBS} install
+
+    cd -
+}
+
 # Install module from GitHub tagged versions or URL
-install_github_module() {
+install_github() {
+    if [ "$1" == "-i" ]
+        then is_ioc=true; shift
+        else is_ioc=false
+    fi
```

Comment:
Please keep the conditional structure we are already using (also use `=` for string comparisons):

```suggestion
    is_ioc=false
    if [ "$1" = "-i" ]; then
        is_ioc=true
        shift
    fi
```